packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -52,6 +52,11 @@ export interface IDropdownProps {
   onRenderOption?: IRenderFunction<IDropdownOption>;
 
   /**
+   * Optional maximum display items in dropdown list.
+   */
+  maxDisplayItems?: number;

```

Comment:
This can be confusing, I see it as we are truncating the options instead of giving a scroll experience, should we name it something like `pageSize`?